test/core/surface/sequential_connectivity_test.cc

```diff
@@ -170,12 +182,16 @@ static grpc_channel* secure_test_create_channel(const char* addr) {
   grpc_channel_credentials* ssl_creds =
       grpc_ssl_credentials_create(test_root_cert, nullptr, nullptr, nullptr);
   grpc_slice_unref(ca_slice);
-  grpc_arg ssl_name_override = {
-      GRPC_ARG_STRING,
-      const_cast<char*>(GRPC_SSL_TARGET_NAME_OVERRIDE_ARG),
-      {const_cast<char*>("foo.test.google.fr")}};
+  const int kNumArgs = 2;
+  grpc_arg args[kNumArgs] = {
+      {GRPC_ARG_STRING,
+       const_cast<char*>(GRPC_SSL_TARGET_NAME_OVERRIDE_ARG),
+       {const_cast<char*>("foo.test.google.fr")}},
+      {GRPC_ARG_INTEGER,
+       const_cast<char*>(GRPC_ARG_USE_LOCAL_SUBCHANNEL_POOL),
+       {.integer = 1}}};
   grpc_channel_args* new_client_args =
-      grpc_channel_args_copy_and_add(nullptr, &ssl_name_override, 1);
+      grpc_channel_args_copy_and_add(nullptr, args, kNumArgs);
```

Comment:
This can also avoid dynamic allocation:

```
grpc_channel_args client_args = { kNumArgs, args };
grpc_channel* channel =
    grpc_secure_channel_create(ssl_creds, addr, &client_args, nullptr);
```